framework/test/jest/integration/utils/storage.js

```diff
@@ -14,103 +14,10 @@
 
 'use strict';
 
-const childProcess = require('child_process');
-const { Storage } = require('../../../../src/components/storage');
-
-// Custom entities
-const {
-	Account,
-	Block,
-	RoundDelegates,
-	Transaction,
-} = require('../../../../src/modules/chain/components/storage/entities');
-
 const {
-	NetworkInfo,
-} = require('../../../../src/modules/network/components/storage/entities');
-
-const {
-	MigrationEntity: Migration,
-} = require('../../../../src/controller/migrations');
-
-const ChainModule = require('../../../../src/modules/chain');
-const NetworkModule = require('../../../../src/modules/network');
-const HttpAPIModule = require('../../../../src/modules/http_api');
-
-const modulesMigrations = {
-	[ChainModule.alias]: ChainModule.migrations,
-	[NetworkModule.alias]: NetworkModule.migrations,
-	[HttpAPIModule.alias]: HttpAPIModule.migrations,
-};
-
-/**
- * @param {string} table
- * @param {Logger} logger
- * @param {Object} storage
- * @param {function} cb
- */
-const clearDatabaseTable = async (storageInstance, table) =>
-	storageInstance.adapter.db.query(`DELETE FROM ${table}`);
-
-class StorageSandbox extends Storage {
-	constructor(dbConfig) {
-		if (!process.env.NODE_ENV || process.env.NODE_ENV !== 'test') {
-			throw new Error(
-				`storage_sandbox is meant to be run in test environment only. NODE_ENV is: ${process.env.NODE_ENV}`,
-			);
-		}
-		super(dbConfig, {});
-
-		process.on('exit', async () => {
-			await this._dropDB();
-		});
-	}
-
-	async bootstrap() {
-		await this._dropDB();
-		await this._createDB();
-		await super.bootstrap();
-
-		this.registerEntity('Account', Account);
-		this.registerEntity('Block', Block);
-		this.registerEntity('Transaction', Transaction);
-
-		// Custom entitties
-		this.registerEntity('Migration', Migration);
-		this.registerEntity('NetworkInfo', NetworkInfo);
-		this.registerEntity('RoundDelegates', RoundDelegates);
-
-		await this._createSchema();
-		return true;
-	}
-
-	async cleanup() {
-		super.cleanup();
-		await this._dropDB();
-	}
-
-	_dropDB() {
-		return new Promise(resolve => {
-			childProcess.exec(`dropdb ${this.options.database}`, () => resolve());
-		});
-	}
-
-	_createDB() {
-		return new Promise((resolve, reject) => {
-			childProcess.exec(`createdb ${this.options.database}`, error => {
-				if (error) {
-					return reject(error);
-				}
-				return resolve();
-			});
-		});
-	}
-
-	async _createSchema() {
-		await this.entities.Migration.defineSchema();
-		await this.entities.Migration.applyAll(modulesMigrations);
-	}
-}
+	StorageSandbox,
+	clearDatabaseTable,
+} = require('../../../mocha/common/storage_sandbox');
```

Comment:
Instead of relying `mocha`, probably it's better to keep `jest` version and remove the mocha one?

Reply:
We can cleanup these stuff later in other issue.

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/4626 @shuse2